packages/device_info/lib/device_info.dart

```diff
@@ -241,6 +244,39 @@ class IosDeviceInfo {
       localizedModel: json['localizedModel'],
       identifierForVendor: json['identifierForVendor'],
       isPhysicalDevice: json['isPhysicalDevice'] == 'true',
+      utsname: new IosUtsname._(
+        sysname: json['utsname.sysname'],
+        nodename: json['utsname.nodename'],
+        release: json['utsname.release'],
+        version: json['utsname.version'],
+        machine: json['utsname.machine'],
+      ),
     );
   }
 }
+
+/// Information derived from `utsname`.
```

Comment:
Maybe also link to http://pubs.opengroup.org/onlinepubs/7908799/xsh/sysutsname.h.html

Reply:
Done